Server/Game/Commands/Misc/HelpCommand.cs

```diff
@@ -0,0 +1,45 @@
+using Platform_Racing_3_Server.Game.Client;
+using Platform_Racing_3_Server_API.Game.Commands;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Platform_Racing_3_Server.Game.Commands.Misc
+{
+    internal class HelpCommand : ICommand
+    {
+        public string Permission => null;
+
+        public void OnCommand(ICommandExecutor executor, string label, ReadOnlySpan<string> args)
+        {
+            if (executor is ClientSession session)
```

Comment:
This should also be available on console

Reply:
:+1: